src/axom/primal/tests/primal_clip.cpp

```diff
@@ -503,44 +525,60 @@ void check_oct_tet_clip(double EPS)
   OctahedronType* oct = axom::allocate<OctahedronType>(1);
   PolyhedronType* res = axom::allocate<PolyhedronType>(1);
 
-  tet[0] = TetrahedronType(PointType {1, 0, 0},
-                           PointType {1, 1, 0},
-                           PointType {0, 1, 0},
-                           PointType {1, 0, 1});
-
-  oct[0] = OctahedronType(PointType {1, 0, 0},
-                          PointType {1, 1, 0},
-                          PointType {0, 1, 0},
-                          PointType {0, 1, 1},
-                          PointType {0, 0, 1},
-                          PointType {1, 0, 1});
+  // Shapes on host
+  TetrahedronType tet_host;
+  OctahedronType oct_host;
+  PolyhedronType res_host;
```

Comment:
Likely outside the scope of this PR, but we should probably have a standardized convention for labeling variables as host, device and views and use them consistently within axom and its tests and examples.

Perhaps we should create a separate issue to track this?

Reply:
Issue #1333 opened to track this issue.